### DISCUSSION TITLE
Comment Plugin that renders into html for seo purposes

### DISCUSSION THREAD

**Discussion by FLMMFL** [GetPublii/Publii#1546] [General]
Hi all, hope your well. 
Do you know of a plugin that will work with Publii that specifically renders the comments in html, so that it becomes useful for SEO?  I assume with most plugins being iframes, they have zero SEO value. Any suggestions would be appreciated

**Comment by bobmitro** [comment]
Unfortunately, most commenting plugins use iframes, which do not contribute to SEO.  Publii is a static site generator, so integrating a comment system directly into the static HTML can be challenging. To achieve this, you would need to use a server-side language like PHP to render comments in HTML, making them useful for SEO.
